src/api.ts

```diff
@@ -1,13 +1,13 @@
 import { MovieData } from './types/MovieData';
 import { ResponseError } from './types/ReponseError';
 
-const API_URL = 'https://www.omdbapi.com/?apikey=your-key';
+const API_URL = 'https://www.omdbapi.com/?apikey=378681db';
 
 export function getMovie(query: string): Promise<MovieData | ResponseError> {
   return fetch(`${API_URL}&t=${query}`)
     .then(res => res.json())
-    .catch(() => ({
+    .catch((error) => ({
       Response: 'False',
-      Error: 'unexpected error',
+      Error: error,
```

Comment:
you can also throw new error here and handle this error in catch {} while calling api function